addons/binding/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/RFXComValueSelector.java

```diff
@@ -64,7 +65,7 @@
     private final String text;
     private Class<? extends Item> itemClass;
 
-    private RFXComValueSelector(final String text, Class<? extends Item> itemClass) {
+    RFXComValueSelector(final String text, Class<? extends Item> itemClass) {
```

Comment:
Nope :-) I'll fix this but I plan to make the fields of RFXComLighting4Message private as well.